src/Command/LoadScreenLayoutsCommand.php

```diff
@@ -37,7 +38,23 @@ final protected function execute(InputInterface $input, OutputInterface $output)
             try {
                 $content = json_decode(file_get_contents($filename), false, 512, JSON_THROW_ON_ERROR);
 
-                $screenLayout = new ScreenLayout();
+                if (isset($content->id)) {
+                    $loadedScreenLayout = $this->entityManager->getRepository(ScreenLayout::class)->findBy(['id' => Ulid::fromString($content->id)]);
```

Comment:
Using `findOneBy` will give you a `ScreenLayout` instance or `null`. Then you can do a simple null check instead of the `is_array`, `count`,  `array_shift`